Mathlib/Topology/Algebra/UniformField.lean

```diff
@@ -89,11 +89,11 @@ theorem continuous_hatInv [CompletableTopField K] {x : hat K} (h : x ≠ 0) :
     rw [eq_bot]
     exact comap_bot
 
-open Classical in
 /-
 The value of `hat_inv` at zero is not really specified, although it's probably zero.
 Here we explicitly enforce the `inv_zero` axiom.
 -/
+open Classical in
```

Comment:
This is moving the comment above the open Classical, to make the linter in #22670 happy. (If this should be a doc comment instead, I'm happy to change that aspect instead.)

Reply:
I'm not certain, but I think this is meant to be a doc comment

Reply:
Sure, then I'll make it one.